index.ts

```diff
@@ -5,5 +5,5 @@ import 'dotenv/config';
 const PORT = process.env.PORT || 5000;
 
 app.listen(PORT, () => {
-  console.log(`🚀 Server is running on port ${process.env.CLIENT_URL}`);
+  console.log(`🚀 Server is running on port ${process.env.PORT}`);
```

Comment:
maybe you can leave the previos link for localhost or add template `http://localhost:${process.env.PORT}`